package.json

```diff
@@ -3,15 +3,21 @@
   "version": "1.3.1",
   "description": "A database viewer for the Shabad OS database",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
     "start": "npm-run-all -l -p start:*",
     "start:backend": "cd backend && npm start",
     "start:frontend": "cd frontend && npm start",
     "build": "cd frontend && npm run build",
     "install-deps": "npm-run-all -l -p install-deps:*",
     "install-deps:frontend": "cd frontend && npm install",
     "install-deps:backend": "cd backend && npm install",
-    "postinstall": "npm run install-deps"
+    "postinstall": "npm run install-deps",
+    "cypress": "cypress open",
```

Comment:
`test:e2e` and `test:e2e:headless`

Reply:
One is for ci jobs and other for running in dev